test/db/cmd/write

```diff
@@ -61,19 +61,18 @@ RUN
 
 NAME=labels replacements (common prefix)
 FILE=malloc://1024
-BROKEN=1
+ARGS=-a x86 -b 32
 CMDS=<<EOF
-e asm.arch=x86
-e asm.bits=32
+p8 7 @ 0
 waf bins/other/asm/multiline_prefix.asm
-p8 7@0
+p8 7 @ 0
 EOF
 EXPECT=<<EOF
-9090e8fbffffff
+00000000000000
+9090e803000000
```

Comment:
this does not look correct. you are compiling code.

Reply:
What do you mean?
It's what `waf` of this code returns:
```
label:
nop
nop
labelA:
call labelA
```